### PR TITLE
#12433

### LINKED ISSUE #12433: flatnonzero

### PR DESCRIPTION
**Pull request by ndurumo254**
close #12433

### CODE REVIEW

**Comment by ndurumo254** [comment]
close #12433

**Comment by ndurumo254** [comment]
##12433

**Comment by ndurumo254** [comment]
#12433

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py [hunk 1]
```diff
@@ -278,3 +278,29 @@ def test_numpy_sort(
         a=x[0],
         axis=axis,
     )
+
+
+# flatnonzero
+@handle_frontend_test(
+    fn_tree="jax.numpy.flatnonzero",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py:287]: We should test for all numeric dtypes I am sure that this function should support it

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey 

Just few changes are needed, will merge then😊

Thanks

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!
Going to merge

**Comment by ndurumo254** [comment]
close #12436

**Pull request merged**